src/components/select/select.js

```diff
@@ -1136,11 +1142,11 @@ function SelectProvider($$interimElementProvider) {
            */
           function mouseOnScrollbar() {
             var clickOnScrollbar = false;
-            if (ev && (ev.currentTarget.children.length > 0)) {
+            if (mouseDownXPosition != null && ev && (ev.currentTarget.children.length > 0)) {
```

Comment:
Agreed that such scenarios are edge cases. I like, however, to always *balance* my event logic: if one (1) action sets a state then a *paired* action should clear that state.